src/tracks/ui/Scrubbing.h

```diff
@@ -149,11 +148,17 @@ class AUDACITY_DLL_API Scrubber final
 private:
    void UpdatePrefs() override;
 
+#ifdef USE_SCRUB_THREAD
+   //! @pre `!mThread.joinable()`
+#endif
```

Comment:
`#ifdef` looks a bit strange here. Probably ```//! @pre `!defined(USE_SCRUB_THREAD) || !mThread.joinable()` ``` would be better readable (and will be visible in docs too!)